cmake/configure/configure_3_petsc.cmake

```diff
@@ -103,6 +103,15 @@ MACRO(FEATURE_PETSC_CONFIGURE_EXTERNAL)
   SET(DEAL_II_EXPAND_PETSC_BLOCKVECTOR "PETScWrappers::BlockVector")
   SET(DEAL_II_EXPAND_PETSC_MPI_VECTOR "PETScWrappers::MPI::Vector")
   SET(DEAL_II_EXPAND_PETSC_MPI_BLOCKVECTOR "PETScWrappers::MPI::BlockVector")
+  #
+  # FIXME:
+  # temporary variable until deal.II fully support complex-valued PETSc
+  IF(NOT PETSC_USE_COMPLEX)
```

Comment:
p.s. i think the default `FIND_PACKAGE(PETSC)` simply does not read all the configure options.